learning_table_tennis_from_scratch/ppo_config.py

```diff
@@ -56,8 +81,9 @@ def from_json(cls, jsonpath):
                 raise ValueError(
                     "failed to find the attribute {} " "in {}".format(s, jsonpath)
                 )
-        return instance.get()
+        return instance
 
+    # FIXME Is this used anywhere?
     @staticmethod
     def default_path():
         return os.path.join(
```

Comment:
Is this `default_path()` method used anywhere?  From searching the repository it seems that no.  If not, I would probably remove the method (the path returned by it does not look to me like something that is used anywhere in our setup).